releases/release-5.0.4.md

```diff
@@ -0,0 +1,169 @@
+---
+title: TiDB 5.0.4 Release Notes
+---
+
+# TiDB 5.0.4 Release Notes
+
+Release Date: September 27, 2021
+
+TiDB version: 5.0.4
+
+## Compatibility Changes
+
++ TiDB
+
+    - Revert #19341 to avoid #24326 [#26258](https://github.com/pingcap/tidb/pull/26258)
+
+## Feature enhancements
+
++ TiDB
+
+- Support set `tidb_enforce_mpp=1` to enforce use mpp mode [#26382](https://github.com/pingcap/tidb/pull/26382)
+
++ TiKV
+
+    - Support changing CDC configs dynamically [#10685](https://github.com/tikv/tikv/pull/10685)
+
++ TiFlash
+
+    - Support the `HAVING()` function in DAG requests
+    - Support the `DATE()` function
+    - Add Grafana panels for write throughput per instance
+
```

Comment:
* PD
  * PD will keep the enough health peers if the region have down/pending peers while the placement rule is enabled
  * PD will dynamically adjust retrying calculating scheduling in order to avoid consuming too much cpu resource
  * improve the performance of synchronization between pd leader and other members
  * PD will only receive json data file for 'persist-file' api considering security
  * improve the performance of calculating hot region scheduling.